### DISCUSSION TITLE
How to use 2 different plugins in one rexx

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#397] [Q&A]
what is the best way to use 2 different plugins in a rexx? They are both in different plugin directories  (also after build in the cmake/build dir).

the rexx looks like this and is in the **system plugin** directory

```
/* System Plugin Test */
options levelb
import system
import fileio
...
```
from here, I can't use the fileio plugin as it is in the **fileio** directory (also in the cmake/build dir). Is there a place where I could  copy all plugins to access them concurrently?

**Comment by adesutherland** [comment]
Does 
`import system fileio`
work?

If you copy the plugins to either the directory of your rexx program or the directory of the rxc binary the compiler should find them all

**Reply by Peter-Jacob** [reply]
no `import system fileio` does no work. 
- I copied **rx_fileio.rxplugin** to the source directory from where I start my **system_test.rexx**. This is where the plugin c source is located. 
- next trial, copied it to: `\...\CREXX\F0049\cmake-build-debug\lib\plugins\system` 
- at last I tried `\...\CREXX\F0049\compiler`
same result

all ended with: 
```
118: Signal 5 (FUNCTION_NOT_FOUND)
118: No Signal Handler not supported [yet!] so Aborting
Failed
```

**Reply by adesutherland** [reply]
Currently rxvm does not search for imports (it was a security idea that I have since abandoned - we *will* have the same logic as rxc in time). Anyway this means you have to specify each file for rxvm i.e.

`rxvm your_program rx_fileio rx_system etc`

Does that work?
